File: mobile/lib/api/models/artist.dart
class Artist {
  final String browseId;
  final String title;
  final String thumbnail;
  final String author;
  final String subscriberText;
  final bool isExplicit;

  const Artist({
    required this.browseId,
    required this.title,
    required this.thumbnail,
    required this.author,
    required this.subscriberText,
    required this.isExplicit,
  });

  factory Artist.fromJson(Map<String, dynamic> json) {
    return Artist(
      browseId: json["browseId"] as String,
      title: json["title"] as String,
      thumbnail: json["thumbnail"] as String,
      author: json["author"] as String,
      subscriberText: json["subscriberText"] as String,
      isExplicit: json["isExplicit"] as bool,
    );
  }
}

File: server/models/api/artist.go
package api_models

type Artist struct {
	BrowseId       string `json:"browseId"`
	Title          string `json:"title"`
	Thumbnail      string `json:"thumbnail"`
	Author         string `json:"author"`
	SubscriberText string `json:"subscriberText"`
	IsExplicit     bool   `json:"isExplicit"`
}
